api/r/cellxgene.census/vignettes/census_datasets.Rmd

```diff
@@ -80,12 +90,32 @@ dataset_seurat <- expt_query$to_seurat(c(counts = "raw"))
 print(dataset_seurat)
 ```
 
-You can also use the `cellxgene.census::get_source_h5ad_uri()` API to fetch a URI pointing to the H5AD associated with this `dataset_id`. This is the same H5AD you can download from the CELLxGENE Portal, and may contain additional data-submitter provided information which was not included in the Census.
+## Downloading the original source H5AD file of a dataset
 
-The "locator" returned by this API will include a URI and additional information that may be necessary to use the URI (eg, the S3 region).
+You can use the `cellxgene.census::get_source_h5ad_uri()` API to fetch a URI pointing to the H5AD associated with this `dataset_id`. This is the same H5AD you can download from [CZ CELLxGENE Discover](https://github.com/chanzuckerberg/cellxgene-census/blob/main/api/python/notebooks/api_demo/census_datasets.ipynb), and may contain additional data-submitter provided information which was not included in the Census.
 
+To do this you can fetch the location in the cloud or directly download to your system.
+
+```{r}
+# Option 1: Direct download
+cellxgene.census::download_source_h5ad("0bd1a1de-3aee-40e0-b2ec-86c7a30c7149",
+  file = "/tmp/Tabula_Muris_Senis-bone_marrow.h5ad",
+  overwrite = TRUE
+)
+```
 ```{r}
+# Option 2: Get location and download via preferred method
 cellxgene.census::get_source_h5ad_uri("0bd1a1de-3aee-40e0-b2ec-86c7a30c7149")
 ```
 
-The `cellxgene.census::download_source_h5ad()` API downloads the H5AD to a local file, which can then be used in R using [SeuratDisk's anndata converter](https://mojaveazure.github.io/seurat-disk/articles/convert-anndata.html).
+The local H5AD file can be used in R using [SeuratDisk's anndata converter](https://mojaveazure.github.io/seurat-disk/articles/convert-anndata.html).
```

Comment:
Just a reminder: once we switch this to point to the versioned artifacts, we'll be able to download RDS files directly, which I believe will be a very nice feature for R users.